src/plugins/auto/plugin.cpp

```diff
@@ -452,12 +478,8 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
         autoSContext->_plugin = this;
         autoSContext->_core = GetCore();
         autoSContext->_LogTag = _LogTag;
-        auto tmpiter = fullConfig.find(ov::intel_auto::device_bind_buffer.name());
-        if (tmpiter != fullConfig.end() && tmpiter->second == PluginConfigParams::YES)
-            autoSContext->_bindBuffer = true;
-        auto tmpiter_enableStartupFallback = fullConfig.find(ov::intel_auto::enable_startup_fallback.name());
-        if (tmpiter_enableStartupFallback != fullConfig.end() && tmpiter_enableStartupFallback->second == PluginConfigParams::NO)
-            autoSContext->_startupfallback = false;
+        autoSContext->_bindBuffer = loadConfig.get_property(ov::intel_auto::device_bind_buffer);
+        autoSContext->_startupfallback = loadConfig.get_property(ov::intel_auto::enable_startup_fallback);
         return std::make_shared<AutoExecutableNetwork>(autoSContext, std::make_shared<AutoSchedule>());
```

Comment:
@WeldonWangwang please help review the change

Reply:
Agree